geonode/templates/_permissions_form.html

```diff
@@ -252,13 +252,19 @@ <h4 class="modal-title" id="exampleModalLabel">{% trans "Message box" %}</h4>
             <div class="modal-body">
 
               <div class="form-group">
-                <div class="input-group input-file" name="zipfile" id="zipfile" data-toggle="tooltip" data-placement="bottom" data-content="Mandatory files : SHP , DBF">
+                <div class="input-group input-file" name="zipfile" id="zipfile" data-toggle="tooltip" data-placement="bottom" 
+                      data-content="{% trans "Mandatory files : SHP , DBF" %}">
                   <span class="input-group-btn">
-                    <button class="btn btn-default btn-choose" type="button">Choose</button>
+                    <button id="geo-limits-upload-btn" class="btn btn-default btn-choose" type="button" 
+                            data-toggle="tooltip" data-placement="bottom"
+                            title="{% trans "Provie a ZIP file containing an ESRI Shapefile. If the ZIP provides also a .prj file, you don't have to specify the EPSG SRID" %}">{% trans "Choose" %}</button>
```

Comment:
I would change "Provie" (which also contains a type) to "Upload", to avoid repetition.